elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base),
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
+  Color: pfe-var(text),
+  FontSize: pfe-var(font-size),
+  Height: 36px,
+  Padding: 8px 6px,
+);
+
+:host {
+  display: inline-block;
+  position: relative;
+}
+
+::slotted(input) {
+  all: unset;
+  opacity: 0;
+  padding: pfe-local(Padding);
+  border-radius: pfe-local(BorderRadius);
+  border: pfe-local(BorderWidth) solid transparent;
+  height: pfe-local(Height) !important;
```

Comment:
Would min-height make sense here?

Reply:
I don't _think_ so? Tell me if I'm wrong. 

The idea is that the light DOM input and the shadow DOM input are the exact same height and the shadow DOM input will lay over the top of the light DOM input. The reason for this is so the native form validation messages from the browser appear in the right spot.

This is a screenshot of the two input fields being the exact same height. 
![image](https://user-images.githubusercontent.com/330256/90822532-9e177a80-e302-11ea-9ba1-fcf79f1f00a8.png)

If the two aren't the same height, you can see how we start running into issues.
![image](https://user-images.githubusercontent.com/330256/90822696-d3bc6380-e302-11ea-8a0f-20f6782adbfc.png)

Reply:
This is stale. We've abandoned the approach of using a light DOM and shadow DOM version stacked on top of each other.